python/dgl/distributed/partition.py

```diff
@@ -1143,6 +1211,9 @@ def get_homogeneous(g, balance_ntypes):
                 val.append(
```

Comment:
Shall we check `if F.sum(F.astype(inner_etype_mask, F.int64), 0) == 0:` here?

Reply:
no, current impl will append `0` and it's expected behavior. the `val` is for sanity check only.